openapi/v2/core-metadata.yaml

```diff
@@ -781,17 +747,21 @@ components:
       properties:
         name:
           type: string
-        get:
-          type: array
-          items:
-            $ref: '#/components/schemas/ResourceOperation'
-        set:
+        isHidden:
+          type: boolean
+          description: "Indicate the visibility of any command via Core Command."
```

Comment:
Change `any command via Core Command` to `the DeviceCommand via a CoreCommand`

Reply:
Modified.